src/main/scripts/src/app/app.component.scss

```diff
@@ -11,7 +11,7 @@ header {
 }
 
 main {
-  width: 100vw;
-  padding: 0 5vw;
-  overflow-y: auto;
+  margin: 10px;
+  height: calc(100vh - 90px);
```

Comment:
But this is the only way I was able to make main content scrollable without scrolling header. Do you know better solution?